static/app/views/alerts/rules/metric/details/sidebar.tsx

```diff
@@ -140,6 +141,21 @@ export function MetricDetailsSidebar({
 
   const ownerId = rule.owner?.split(':')[1];
   const teamActor = ownerId && {type: 'team' as Actor['type'], id: ownerId, name: ''};
+  let conditionType;
+  switch (
+    rule.monitorType === MonitorType.ACTIVATED &&
+    typeof rule.activationCondition !== 'undefined' &&
+    rule.activationCondition
```

Comment:
thoughts on pulling this out as a variable? It might read a little more cleanly if you have something like:
```ts
const activationConditionType = rule.monitorType === MonitorType.ACTIVATED &&
    typeof rule.activationCondition !== 'undefined' &&
    rule.activationCondition;
    
switch(activationConditionType) {
  ...
}
```
(or maybe it could just be an if statement? 🤔)

Reply:
🤷 
I'll keep a switch statement since this could potentially become more than just these 2
fine wiht breaking out into a const var